### ISSUE TITLE
Improper Model Training

### ISSUE THREAD

**Issue by PranavU-Coder** [PixelHalide/rankpredictor#4]
predictor_scripts.js doesn't really have any validation set on which the model was evaluated on
 
how can this be any standard for a prediction model ? we need data augmentation on newly - collected data after performing EDA then split it for our use case

if possible , please grant me the  access to codebase so we can convert this to a long-term project with proper roadmap (referring to the github's projects tool and branch protection rule for preserving the codebase)

my techstack : 

> html + css + js
> react 
> python + numpy + pandas + matplotlib + scikit-learn + tensorflow and keras api

my experience : 

> worked with MIT-B AUV Team for DL-Models and other robotics-related work (6 months)
> leading Manipal OSF (4 months)
> currently a contributor on kaggle
